File: app/src/main/java/com/chooloo/www/callmanager/ui/contact/ContactBottomDialogFragment.java
package com.chooloo.www.callmanager.ui.contact;

import android.os.Bundle;

import com.chooloo.www.callmanager.entity.Contact;
import com.chooloo.www.callmanager.ui.base.BaseBottomSheetDialogFragment;

public class ContactBottomDialogFragment extends BaseBottomSheetDialogFragment {

    public static final String TAG = "contact_bottom_dialog_fragment";
    public static final String ARG_CONTACT = "contact";

    private ContactFragment mContactFragment;

    public static ContactBottomDialogFragment newInstance(Contact contact) {
        Bundle args = new Bundle();
        args.putSerializable(ARG_CONTACT, contact);
        ContactBottomDialogFragment fragment = new ContactBottomDialogFragment();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    public void onSetup() {
        mContactFragment = ContactFragment.newInstance((Contact) getArgsSafely().getSerializable(ARG_CONTACT));
        putFragment(mContactFragment);
    }
}

File: app/src/main/java/com/chooloo/www/koler/ui/contact/ContactBottomDialogFragment.kt
package com.chooloo.www.koler.ui.contact

import android.os.Bundle
import com.chooloo.www.koler.ui.base.BaseBottomSheetDialogFragment
import com.chooloo.www.koler.ui.contact.ContactFragment.Companion.ARG_CONTACT_ID

class ContactBottomDialogFragment : BaseBottomSheetDialogFragment() {
    private val _contactId by lazy { argsSafely.getLong(ARG_CONTACT_ID) }
    private val _contactFragment by lazy { ContactFragment.newInstance(_contactId) }

    companion object {
        const val TAG = "contact_bottom_dialog_fragment"

        fun newInstance(contactId: Long) = ContactBottomDialogFragment().apply {
            arguments = Bundle().apply {
                putSerializable(ARG_CONTACT_ID, contactId)
            }
        }
    }

    override fun onSetup() {
        putFragment(_contactFragment)
    }
}